clinica/iotools/bids_utils.py

```diff
@@ -423,62 +448,68 @@ def create_scans_dict(
     return scans_dict
 
 
-def write_modality_agnostic_files(study_name, bids_dir):
-    """
-    Write the files README, dataset_description.json, .bidsignore and .bids-validator-config.json
-    at the root of the BIDS directory.
+def _write_bids_dataset_description(
+    study_name: str,
+    bids_dir: Union[str, Path],
+    bids_version: Optional[str] = None,
+) -> None:
+    """Write `dataset_description.json` at the root of the BIDS directory."""
+    from clinica.iotools.bids_dataset_description import BIDSDatasetDescription
 
-    Args:
-        study_name: name of the study (Ex ADNI)
-        bids_dir: path to the bids directory
-    """
-    import json
-    from os import path
+    if bids_version:
+        bids_desc = BIDSDatasetDescription(name=study_name, bids_version=bids_version)
+    else:
+        bids_desc = BIDSDatasetDescription(name=study_name)
+    with open(Path(bids_dir) / "dataset_description.json", "w") as f:
+        bids_desc.write(to=f)
 
-    import clinica
-    from clinica.iotools.bids_dataset_description import BIDSDatasetDescription
 
-    with open(path.join(bids_dir, "dataset_description.json"), "w") as f:
-        BIDSDatasetDescription(name=study_name).write(to=f)
+def _write_readme(bids_dir: Union[str, Path]) -> None:
+    """Write `README`file at the root of the BIDS directory."""
+    import clinica
 
-    with open(path.join(bids_dir, "README"), "w") as f:
+    with open(Path(bids_dir) / "README", "w") as f:
         f.write(
             f"This BIDS directory was generated with Clinica v{clinica.__version__}.\n"
             f"More information on http://www.clinica.run\n"
```

Comment:
Let me suggest to use secure URLs when possible.

```suggestion
            f"More information on https://www.clinica.run\n"
```